library/alloc/src/collections/btree/map/entry.rs

```diff
@@ -269,6 +269,31 @@ impl<'a, K: Ord, V, A: Allocator + Clone> Entry<'a, K, V, A> {
             Vacant(entry) => Vacant(entry),
         }
     }
+
+    /// Sets the value of the entry, and returns an `OccupiedEntry`.
```

Comment:
Suggestion: maybe intra-doc link?

Reply:
Rustdoc will already have that link in the actual function signature. Is it useful to link it again?

e.g. on `HashMap`: https://doc.rust-lang.org/beta/std/collections/hash_map/enum.Entry.html#method.insert_entry

Reply:
Ah good point, nevermind then 👍